cert/lib/cert/runner.rb

```diff
@@ -160,11 +160,18 @@ def create_certificate
       request_path = File.expand_path(File.join(Cert.config[:output_path], "#{certificate.id}.certSigningRequest"))
       File.write(request_path, csr.to_pem)
 
-      private_key_path = File.expand_path(File.join(Cert.config[:output_path], "#{certificate.id}.p12"))
+      private_key_path = File.expand_path(File.join(Cert.config[:output_path], "#{certificate.id}.pkey"))
```

Comment:
We can't do that, as this would be a breaking change for setups that assume the `.p12` file extension. Or does https://github.com/fastlane/fastlane/pull/8440/files#diff-b4b9b8ba735c52a8fe2212a8f0981c4dR171 write the file on the same output location?